src/logic/calculate.js

```diff
@@ -0,0 +1,18 @@
+import operate from './operate';
+
+const calculate = (buttonName, data) => {
+  const { next, operation, total } = data;
+  const result = operate(
```

Comment:
I believe you want to add the conditions in the next milestone.

Heads-Up:
Make sure to return an error message when a number is divided by 0. `(e.g. 4 / 0)`